docs/natural-history-inputs.md

```diff
@@ -0,0 +1,95 @@
+# A central natural history manager
+
+## Overview
+We provide a way for reading in user-specified natural history parameters (infectiousness over time/the
+generation interval, viral load over time, etc.) and parameterizing transmission based on these parameters.
+In the future, the input CSV can be expanded to also include symptom onset and improvement times as part of
+the natural history parameter set.
+
+A natural history parameter set consists of a set of values of all the natural history parameters -- the
+generation interval over time, the viral load, the symptom onset and improvement times, the time to
+hospitalization, etc. There may be correlations between these parameters -- so that higher viral loads
+are associated with longer times to symptom improvement -- or there may not be. By having the user specify
+a CSV file which contains all of these parameter sets, not only does this mean that the user determines
+the distribution of each of these parameters outside of Rust, but the user is also able to impose any
+meaningful correlations between parameters.
+
+## Data input format
+
+The input CSV includes samples of the natural history parameters. For now, we are interested in a single
+natural history parameter: the generation interval CDF over time. Data must be input in a long format, as
+such:
+
+| id | time | gi_cdf |
+| --- | --- | --- |
+| 0 | 0.0 | 0.0 |
+| 0 | 1.0 | 0.5 |
+| 0 | 2.0 | 1.0 |
+| 1 | 0.0 | 0.0 |
+| 1 | 1.0 | 0.3 |
+| ... | ... | ... |
+
+The `id` is an identifier that marks a distinct sample from the natural history parameters at a given `time`
+since the person was first infected. The `gi_cdf` describes the fraction of infectiousness that has occured
+in this parameter set a given time. Additional columns like `viral_load` and other time-varying quantities
+can easily be added in this input schema.
```

Comment:
Thanks. Could you help me understand why separate CSVs would be better here? I think Guido and I were against separate CSVs because that could mean that a user could update one CSV but just forget to update the other. We wanted to try to prevent that. Also, if we make an ixa helpers Python package, that could have a function that stitches together multiple incoming CSVs from the user into one main file to feed to ixa.

Reply:
Sorry, I was actually thinking about lines 38 below where the variables were constant in time